keep/api/core/db.py

```
@@ -3134,6 +3083,32 @@
         return incidents
 
 
+def enrich_alerts_with_incidents(tenant_id: str, alerts: List[Alert], session: Optional[Session]=None):
```

Ruff (0.8.0)

`Alert` may be undefined, or defined from star imports

(F405)